### DISCUSSION TITLE
Stop forcing "for you" feed

### DISCUSSION THREAD

**Discussion by slembcke** [community/community#13889] [Other Feature Feedback, Questions, & Ideas]
You keep auto-selecting the "for you" feed. It's not helpful. I don't want another social network. Stop it.

**Comment by transcental** [comment]
They don't....

If you select the default feed, the next time you open GH it will open on the default feed.

**Reply by slembcke** [reply]
Then they need to fix some bugs because it's happened several times already, and twice today.

**Reply by SleeplessByte** [reply]
Yep. Encoutered the exact same thing. https://github.com/github/feedback/discussions/13885

**Comment by henrypinkard** [comment]
Would be nice to have an option to disable "for you" entirely. I find this feature intrusive and distracting

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
